packages/celotool/README.md

```diff
@@ -1,5 +1,7 @@
 # Celotool
 
```

Comment:
This package was at some point being used to faucet accounts on testnets. It might be worth checking with devops if this is still the case.

Reply:
Will do 👍 

I already heard form majority of people that celotool is not used anymore at all and it's great candidate for deprecation, but will check anyway with someone from devops team 😁

Reply:
After discussions with devops - this is not used by anyone at this stage.
Even for port forwarding this tool is outdated and recommended is manual approach described on devops slack channel 😁